charts/tornjak/templates/NOTES.txt

```diff
@@ -17,6 +17,24 @@ Universal Trusted Workload Identity Service has completed.
       MY_DISCOVERY_DOMAIN: {{ .Values.OIDC.MY_DISCOVERY_DOMAIN }}
 {{- end }}
 
+{{- if .Values.multiCluster.remoteClusters }}
+      Multi-cluster support enabled.
+      Make sure `kubeconfig` secret is created to support a following:
+
+      clusters = {
+          "{{ .Values.clustername }}" = {
+              # use_token_review_api_validation = true
+              service_account_whitelist = ["spire:spire-agent"]
+          },
+      {{- range $k, $v := .Values.multiCluster.remoteClusters }}
+          "{{ $v.name }}" = {
+              service_account_whitelist = ["{{ $v.namespace | default "spire" }}:{{ $v.serviceAccount | default "spire-agent" }}"]
+              kube_config_file = "/tmp/kubeconfigs/{{ $v.name }}"
```

Comment:
mount path should not be `/tmp` since the behavior of `/tmp` is assumed to be transient.

Reply:
Very good point. It should be `/run/spire/kubeconfigs`